src/coreclr/inc/corinfo.h

```diff
@@ -2712,6 +2712,9 @@ class ICorStaticInfo
             CORINFO_SIG_INFO*           sig,            /* IN */
             CORINFO_ARG_LIST_HANDLE     args,           /* IN */
             CORINFO_CLASS_HANDLE       *vcTypeRet       /* OUT */
+#if defined(TARGET_LOONGARCH64)
```

Comment:
This file defines JIT/EE interface. The JIT/EE interface cannot have platform-specific ifdefs. It breaks cross-compilation. If you need to make changes on the JIT/EE interface, the change has to be made for all platforms.

It may be a good idea to factor out the JIT/EE interface updates into a separate PR. There is quite a bit of ceremony required for JIT/EE changes. See https://github.com/dotnet/runtime/blob/main/docs/project/updating-jitinterface.md for details.

Reply:
Yes, I agree with you.
I think the interface liking the ABI-info between JIT and EE maybe need amend.
Now the AMD-ABI has a platform-specific interace, I think it's best that designing a common interface doing that.

Reply:
I have moved the releated code to the following link,
https://github.com/dotnet/runtime/pull/62893

Reply:
https://github.com/dotnet/runtime/pull/62893#discussion_r770293568